### PR TITLE
Wizard: Staking deposit and stake

### PR DESCRIPTION
**Pull request by kimpers**
I apologize for the messy PR, upgrading to the V3 modules did not go as smooth as I was hoping for. I started fixing some of the breaking changes but there was just too many things to fix in order to get the Portal working again so I had to resort to disabling it for now.

Hopefully we can circle back to this and get Portal working with V3.

### CODE REVIEW

File: ts/utils/order_utils.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    async getOrderHashHexAsync(order: Order): Promise<string> {
+        // HACK(kimpers): This only needs to access the pure function getOrderHash and does not need on-chain data
+        const devUtilsContract = new DevUtilsContract('0x0000000000000000000000000000000000000000', {
+            isEIP1193: true,
+        } as any);
+        return devUtilsContract
+            .getOrderHash(order, new BigNumber(order.chainId, 10), order.exchangeAddress)
+            .callAsync();
+    },
+};
```
Comment (kimpers) [inline ts/utils/order_utils.ts:64]: I started migrating logic to use these replacements utilities that are no longer exported in the 0x modules but could not finish it. Leaving this in as a reference for when we want to try to get Portal back on.

File: ts/pages/instant/config_generator.tsx [hunk 2]
```diff
@@ -201,8 +201,9 @@ export class ConfigGenerator extends React.Component<ConfigGeneratorProps, Confi
         if (value.orderSource !== undefined && _.isString(value.orderSource)) {
             this.setState({ isLoadingAvailableTokens: true });
             const networkId = constants.NETWORK_ID_MAINNET;
-            const sraOrderProvider = new StandardRelayerAPIOrderProvider(value.orderSource, networkId);
-            const etherTokenAddress = getContractAddressesForNetworkOrThrow(networkId).etherToken;
+            // TODO(kimpers): is this correct??
+            const sraOrderProvider = new StandardRelayerAPIOrderProvider(value.orderSource);
```
Comment (kimpers) [inline ts/pages/instant/config_generator.tsx:205]: @fragosti is this fine without chain/network id?

**Review by johnrjj — APPROVED** [review]
LGTM. We can circle back on Portal later after staking.

**Comment by kimpers** [comment]
I'll merge this, feel free to continue reviewing and then I can fix it in a follow up PR. Just so that @johnrjj is not blocked waiting for this.

**Pull request merged**